strawberry_django/resolvers.py

```diff
@@ -48,9 +48,10 @@ def get(self, id):
 
     def create(self, data):
         model = self.get_model()
-        instance = model(**data)
-        instance.save()
-        return instance
+        instances = [model(**i) for i in data]
+        for instance in instances:
+            instance.save()
```

Comment:
I'd prefer bulk creation. It's more efficient. What do you think?

https://docs.djangoproject.com/en/3.1/ref/models/querysets/#bulk-create

Reply:
Tried, this method can't return the ID field